concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-505			505	ironwood tree	Capell1956[128]
+Capell-1956-558-506			506	coolibah tree	Capell1956[128]
+Capell-1956-558-507			507	wattle	Capell1956[128]
+Capell-1956-558-508	3243	CYPRESS	508	cypress pine	Capell1956[129]
+Capell-1956-558-509			509	milk tree	Capell1956[129]
+Capell-1956-558-510	469	PANDANUS	510	pandanus	Capell1956[129]
+Capell-1956-558-511			511	paperbark	Capell1956[129]
+Capell-1956-558-512			512	sandalwood	Capell1956[129]
+Capell-1956-558-513			513	stringybark	Capell1956[130]
+Capell-1956-558-514			514	white gum	Capell1956[130]
+Capell-1956-558-515	409	FORKED BRANCH	515	forked branch	Capell1956[130]
+Capell-1956-558-516			516	woolybut	Capell1956[130]
+Capell-1956-558-517	2692	MATERNAL UNCLE (MOTHER'S BROTHER)	517	uncle (m.b.)	Capell1956[131]
+Capell-1956-558-518	2691	PATERNAL AUNT (FATHER'S SISTER)	518	uncle (f.si.)	Capell1956[131]
+Capell-1956-558-519	1363	VOICE	519	voice	Capell1956[131]
+Capell-1956-558-520	1278	VOMIT	520	vomit	Capell1956[131]
+Capell-1956-558-521	1837	VULVA	521	vulva	Capell1956[132]
+Capell-1956-558-522	1311	VERY	522	very	Capell1956[132]
+Capell-1956-558-523	1590	WAIT (FOR)	523	wait for	Capell1956[132]
+Capell-1956-558-524	1602	WAKE UP	524	wake up	Capell1956[132]
+Capell-1956-558-525	1443	WALK	525	walk	Capell1956[133]
+Capell-1956-558-526	165	WALLABY	526	wallaby	Capell1956[133]
+Capell-1956-558-527	1453	WASH	527	wash	Capell1956[133]
+Capell-1956-558-528	948	WATER	528	water	Capell1956[133]
+Capell-1956-558-529			529	waterhole	Capell1956[134]
+Capell-1956-558-530	978	WAVE	530	wave (noun)	Capell1956[134]
+Capell-1956-558-531	3544	WAVE (VERB)	531	wave (vb.)	Capell1956[134]
+Capell-1956-558-532	1601	WEAK	532	weak	Capell1956[134]
+Capell-1956-558-533	976	WEAR	533	wear	Capell1956[135]
+Capell-1956-558-534	1817	WEST	534	west	Capell1956[135]
+Capell-1956-558-535	1236	WHAT	535	what?	Capell1956[135]
+Capell-1956-558-536	1238	WHEN	536	when?	Capell1956[135]
+Capell-1956-558-537	1237	WHERE	537	where?	Capell1956[135]
+Capell-1956-558-538	1025	WHISTLE	538	whistle	Capell1956[136]
+Capell-1956-558-539	1335	WHITE	539	white	Capell1956[136]
+Capell-1956-558-540	883	TERMITE	540	white ant	Capell1956[136]
+Capell-1956-558-541	2839	WHITE MAN	541	white man	Capell1956[136]
+Capell-1956-558-542	3740	WHIRLWIND	542	whirlwind	Capell1956[136]
+Capell-1956-558-543	1235	WHO	543	who?	Capell1956[137]
+Capell-1956-558-544	1270	WHY	544	why?	Capell1956[137]
+Capell-1956-558-545	1199	WIFE	545	wife	Capell1956[137]
+Capell-1956-558-546	960	WIND	546	wind	Capell1956[137]
+Capell-1956-558-547	962	WOMAN	547	woman	Capell1956[138]
+Capell-1956-558-548			548	wood (dry)	Capell1956[138]
+Capell-1956-558-549			549	wood (wet of green)	Capell1956[138]
+Capell-1956-558-550	1390	WRONG	550	wrong	Capell1956[138]
+Capell-1956-558-551	799	WRIST	551	wrist	Capell1956[139]
+Capell-1956-558-552	1257	WING	552	wing	Capell1956[139]
+Capell-1956-558-553	410	YAM	553	yam	Capell1956[139]
+Capell-1956-558-554	339	DIGGING STICK	554	yam-stick	Capell1956[139]
```

Comment:
Similarly to throwing stick and BOOMERANG, is a yam-stick always for digging? Probably an unmap

Reply:
I found definitions of digging stick on the internet that mentions yam stick as an alternative name for it. Do you want me to unmap it anyways?

Reply:
Then you can leave it, thanks for the research